sql/telemetry/fenix_events_v1/view.sql

```diff
@@ -29,6 +29,6 @@ SELECT
       STRUCT(client_info.architecture AS arch)
     ) AS user_properties
 FROM
-    `moz-fx-data-shared-prod.org_mozilla_fenix_live.events_v1`
+    `moz-fx-data-shared-prod.org_mozilla_fenix_stable.events_v1`
```

Comment:
@fbertsch This view is used for amplitude event export. I'm guessing the reference to live table here is either a mistake or a historical artifact. Any objection to switching to stable tables for this?

In particular, I believe the amplitude export job checks to make sure the count is non-zero from this query before running. That check seems meaningless on top of live tables.

Reply:
Yup, that seems correct to me. This probably existed for the same reason as the other one; that there were no stable tables.